example/logstash/2-ecs-conversion.conf

```diff
@@ -16,17 +18,11 @@ filter {
     uppercase => [ '[log][level]' ]
     rename => { '[eventid]' => '[event][id]' }
     convert => { '[event][id]' => 'string' }
-    convert => { '[successful]' => 'boolean' }
     rename => { '[hostip]' => '[host][ip]' }
   }
   if [host][ip] {
     ruby {
       code => "event.set('[host][ip]', Array(event.get('[host][ip]')) )"
     }
   }
-  if [process][args] {
-    ruby {
-      code => "event.set('[process][args]', Array(event.get('[process][args]')) )"
-    }
-  }
```

Comment:
Oh no, I think in PR #8 we lost the ability to adjust fields in place.

Let's not hold off this PR for this. I filed a different issue: https://github.com/elastic/ecs-mapper/issues/16